test/controllers/charts_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
```

Comment:
Likewise with the other comments, we won't be using this test setup, but don't worry about cause we'll delete this stuff later.